### ISSUE TITLE
GPU requirement

### ISSUE THREAD

**Issue by yuzhang03** [Adonis-galaxy/WorDepth#2]
Thanks for sharing the code! I am wondering what is the GPU requirement for reproducing the results in this paper?

**Comment by Adonis-galaxy** [comment]
Hi, the default configuration is for training WorDepth on one RTX 3090 GPU. If the memory of the GPU you used is not enough, you could decrease the batch size and do gradient accumulation to simulate the same batch size.
